mlflow/tracking/fluent.py

```diff
@@ -127,6 +127,37 @@ def start_run(run_id=None, experiment_id=None, run_name=None, nested=False):
     :param nested: Controls whether run is nested in parent run. ``True`` creates a nest run.
     :return: :py:class:`mlflow.ActiveRun` object that acts as a context manager wrapping
              the run's state.
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
```

Comment:
Should we include other usages

usage 1

```
with mlflow.start_run():
    ...
```

usage 2

```
mllfow.start_run()
...
mlflow.end_run()
```

Reply:
This may bloat the code snippets unnecessarily. We do mention at the outset two usages, so I feel that should suffice. I lean more toward the context manager syntax: it's more clean and clear.

Reply:
makes sense.